### PR TITLE
Info banner controller accounts

### LINKED ISSUE #5069: Remove references to Controller Account in the Wiki and Video Tutorials
Add an info note here https://wiki.polkadot.network/docs/learn-controller liking to the discussion on deprecation of Controller Accounts https://forum.polkadot.network/t/staking-controller-deprecation-plan-staking-ui-leads-comms/2748 . Also, unlist the staking video tutorials with references to the controller account on YouTube.

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

**Comment by filippoweb3** [comment]
@DrW3RK I also unlisted one video about controller accounts on the tech explainers playlist

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-validate-polkadot.md:1]: The image in bond DOT section needs replacement as it shows controller account

image

**Review by DrW3RK — CHANGES_REQUESTED** [review]

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md
Comment (filippoweb3) [reply docs/maintain/maintain-guides-how-to-validate-polkadot.md:1]: @DrW3RK I updated the figures

File: docs/learn/learn-controller.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ keywords: [nominated proof of stake, staking, controller]
 slug: ../learn-controller
 ---
 
+:::info Controller accounts are deprecated
+
+Controller accounts are deprecated, for more information see
+[this discussion](https://forum.polkadot.network/t/staking-controller-deprecation-plan-staking-ui-leads-comms/2748).
```
Comment (profradha) [inline docs/learn/learn-controller.md]:
```suggestion
Controller accounts are deprecated. For more information, see
[this discussion](https://forum.polkadot.network/t/staking-controller-deprecation-plan-staking-ui-leads-comms/2748).
```

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md [hunk 2]
```diff
@@ -525,6 +525,13 @@ two accounts and make sure each of them have at least enough funds to pay the fe
 transactions. Keep most of your funds in the stash account since it is meant to be the custodian of
 your staking funds.
 
+:::info Controller accounts are deprecated
+
+Controller accounts are deprecated, for more information see
+[this discussion](https://forum.polkadot.network/t/staking-controller-deprecation-plan-staking-ui-leads-comms/2748).
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
Controller accounts are deprecated. For more information, see
[this discussion](https://forum.polkadot.network/t/staking-controller-deprecation-plan-staking-ui-leads-comms/2748).
```

**Review by DrW3RK — APPROVED** [review]
I found a few other instances on the Wiki where Controller accounts were mentioned. If possible, please do another check just to be sure. Approved to be merged.

**Pull request merged**